modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -385,6 +385,11 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         /** Returns true if there are no layers in the network. */
         CV_WRAP bool empty() const;
 
+        /** @brief Dump net structure, hyperparameters, backend, target and fusion to dot file
+         *  @param path   path to output file with .dot extension
+         *  To see correct backend, target and fusion run after forward().
+         */
+        void dumpNet(String path);
```

Comment:
For testing purposes it is better to dump into string as primary functionality.

Storing into files can be an additional method.

---

> String

Should be `const String&` for input parameters.

Reply:
done